index.html

```diff
@@ -4,18 +4,169 @@
 <head>
     <meta charset="UTF-8">
 
-    <title>Untitled</title>
+    <title>Smashing Magazine - HeatMap</title>
 
     <link rel="stylesheet" href="css/styles.css">
 
+    <link href="https://fonts.googleapis.com/css?family=Fira+Sans:200,500,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,700&display=swap" rel="stylesheet">
+
 </head>
 
 <body>
     <header>
-
+        <div class="header-cont bordered"></div>
+        <div class="header-cont bordered"></div>
+        <div class="header-cont bordered"></div>
+        <div class="header-cont bordered"></div>
+        <div class="header-cont "></div>
     </header>
 
     <main>
+        <div class="title-div">
+            <h2>Don't Miss These Latest <br>Articles</h2>
+        </div>
+        <div class="article-div">
+            <div class="text-div">
+                <h3>Daniel Matheus <i>wrote</i></h3>
+
+                <h1>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h1>
+
+                <p>Leave a lorem ipsum</p>
+                <br><br>
+                <p>#lorem #ipsum</p>
+            </div>
+        </div>
+        <div class="article-div">
+            <div class="text-div">
+                <h3>Daniel Matheus <i>wrote</i></h3>
+
+                <h1>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h1>
+
+                <p>Leave a lorem ipsum</p>
+                <br><br>
+                <p>#lorem #ipsum</p>
+            </div>
+        </div>
+        <div class="article-div">
+            <div class="text-div">
+                <h3>Daniel Matheus <i>wrote</i></h3>
+
+                <h1>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</h1>
+
+                <p>Leave a lorem ipsum</p>
+                <br><br>
+                <p>#lorem #ipsum</p>
+            </div>
+        </div>
+
+        <div class="membership-cont">
```

Comment:
This whole `<div>` tag should also be represented as a section, within that section we should have two `<article>` tags, one which is two thirds in width and should be positioned to the left of the section with just a title and a small paragraph, and the other part we can go by with just the dark gray box u used or make something more resemblant of the site, your choice.